spring-cloud-alibaba-nacos-discovery/src/main/java/org/springframework/cloud/alibaba/nacos/NacosDiscoveryProperties.java

```diff
@@ -381,6 +396,11 @@ public void overrideFromEnv(Environment env) {
 			this.setEndpoint(
 					env.resolvePlaceholders("${spring.cloud.nacos.discovery.endpoint:}"));
 		}
+		if (StringUtils.isEmpty(this.getGroupName())) {
+			this.setEndpoint(
```

Comment:
It seems should invoke this.setGroupName(...)

Reply:
My bad! pushed it again